lib/streamlit/config.py

```diff
@@ -544,6 +544,15 @@ def _browser_server_port():
     return get_option("server.port")
 
 
+@_create_option("browser.wideMode", type_=bool)
+def _browser_wide_mode():
+    """Make wide mode sticky.
+
+    Default: False
+    """
+    return False
+
+
```

Comment:
Since it's defaulting to False, this looks simpler? 

```suggestion
_create_option(
    "browser. wideMode",
    description="""
        Makes wide mode sticky
        Default: False
    """,
    default_val=False,
    type_=bool,
    scriptable=True,
)
```

Also from the [docs](https://docs.streamlit.io/en/latest/api.html?highlight=config#streamlit.set_option), it looks like adding `scriptable=True` allows users to change the setting from within their app which I think would be much nicer than changing from command line. Would have to update docs for `st.set_option` as well.